src/routing_table/mod.rs

```diff
@@ -1141,6 +1141,7 @@ mod tests {
         let known_neighbour = unwrap!(known_neighbour);
         assert!(!table.our_prefix.matches(&known_neighbour));
 
+        // TODO: this can fail since `table.iter()` excludes own_name; maybe it shouldn't?
```

Comment:
Bart already has a fix in #1280:
https://github.com/maidsafe/routing/pull/1280/commits/7d954ead027ea1cbe86d3518a32979afe1eea017